src/Fantomas.Tests/TokenParserTests.fs

```diff
@@ -658,3 +658,29 @@ let isUnix =
 
     getDefines source
     == [ "NETSTANDARD1_6"; "NETSTANDARD2_0" ]
+
+
+
+[<Test>]
+[<Category("Oracle")>]
+let ``ifdef for inline keyword`` () =
+
+    formatSourceString
```

Comment:
Fantomas will format this code twice: once with no defines and once with `DEBUG`.
It might be easier to solve one case at a time by using
```fsharp
[<Test>]
[<Category("Oracle")>]
let ``ifdef for inline keyword, no defines`` () =
    formatSourceStringWithDefines
        []
        """
    let 
#if !DEBUG
        inline
#endif
        map f ar = Async.map (Result.map f) ar
"""
        config
    |> should
        equal
        """
    let 
#if !DEBUG
        inline
#endif
        map f ar = Async.map (Result.map f) ar
"""
```